### ISSUE TITLE
Show other symbols on same parent

### ISSUE THREAD

**Issue by amos402** [amos402/vscode-scope-bar#2]
Also need to add a key binding for helping that check other symbols under parent easily.

**Comment by robofearth** [comment]
Same issue, if a symbol is within another symbol, e.g. JavaScript functions that are wrapped within anonymous functions, only the parent anonymous function is detected/shown.

**Comment by robofearth** [comment]
Perhaps it could have an option to show innermost symbol?

**Comment by amos402** [comment]
![default](https://user-images.githubusercontent.com/3406561/46776372-daa8e100-cd3d-11e8-9ff5-d26a559200bf.png)
You mean like this? But the anonymous function does not have a symbol name in fact, it's hard to navigate to the function.

**Comment by robofearth** [comment]
That's true. I was thinking of a way to distinguish between a global wrapping anonymous function, like so:

<img width="329" alt="screen shot 2018-10-11 at 9 31 08 pm" src="https://user-images.githubusercontent.com/8605335/46798290-3e162b80-cd9d-11e8-990f-390151cab9ec.png">

<img width="322" alt="screen shot 2018-10-11 at 9 31 00 pm" src="https://user-images.githubusercontent.com/8605335/46798299-44a4a300-cd9d-11e8-95e9-80c8c79bab19.png">

**Comment by robofearth** [comment]
An update on this. I found that VS Code has a similar feature built-in, but disabled by default:

![image](https://user-images.githubusercontent.com/8605335/48606240-3e749880-e9d3-11e8-9975-9f0d94142433.png)

When enabled, shows this:

![image](https://user-images.githubusercontent.com/8605335/48606250-492f2d80-e9d3-11e8-99ec-0ff6f8f8a70c.png)

Not sure if this works for anyone else, but solves the issue I was having using this extension. This issue can now be closed.

Edit: Typos
